### PR TITLE
Add remove-1 test

### PR DESCRIPTION
**Pull request by j-mracek**
It test case when dnf removes dependency of package that should be removed from
transaction. It should result in fail and no package should be removed.

### CODE REVIEW

File: dnf-docker-test/features/remove-1.feature [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        And I successfully run "dnf -y install TestA"
+      Then rpmdb changes are
+        | State     | Packages     |
+        | installed | TestA, TestB |
+
+  Scenario: dnf remove (when there are such pkgs)
+       When I save rpmdb
+        And I run "dnf -y remove TestB -x TestA"
+       Then the command should fail
+        And rpmdb changes are
```
Comment (ignatenkobrain) [inline dnf-docker-test/features/remove-1.feature]: `rpmdb doesn't change` IIRC

**Comment by j-mracek** [comment]
@ignatenkobrain Fixed

**Comment by jrohel** [comment]
@dnf-bot r+

**Comment by jrohel** [comment]
@rh-atomic-bot r+

**Pull request merged**